srcpkgs/synapse/template

```diff
@@ -11,21 +11,22 @@ depends="python3-jsonschema python3-frozendict python3-canonicaljson
  python3-phonenumbers python3-prometheus_client python3-attrs python3-netaddr
  python3-setuptools python3-bleach python3-Jinja2 python3-psycopg2 python3-lxml
  python3-saml2 python3-treq python3-macaroons python3-sortedcontainers
- python3-typing_extensions python3-ijson"
-checkdepends="$depends python3-parameterized python3-mock"
+ python3-typing_extensions python3-ijson python3-matrix-common"
```

Comment:
python3-matrix-common should be in $depends according to https://github.com/matrix-org/synapse/blob/84bfe47b01402cf037417f0026ed4077223ed259/synapse/python_dependencies.py#L91

Reply:
My bad, I thought this line was in checkdepends